packages/tempus/src/Tempus_IntegratorObserverSubcycling_impl.hpp

```diff
@@ -0,0 +1,95 @@
+// @HEADER
+// ****************************************************************************
+//                Tempus: Copyright (2017) Sandia Corporation
+//
+// Distributed under BSD 3-clause license (See accompanying file Copyright.txt)
+// ****************************************************************************
+// @HEADER
+
+#ifndef Tempus_IntegratorObserverSubcycling_impl_hpp
+#define Tempus_IntegratorObserverSubcycling_impl_hpp
+
+#include "Tempus_Stepper.hpp"
+
+namespace Tempus {
+
+template<class Scalar>
+IntegratorObserverSubcycling<Scalar>::IntegratorObserverSubcycling(){}
+
+template<class Scalar>
+IntegratorObserverSubcycling<Scalar>::~IntegratorObserverSubcycling(){}
+
+template<class Scalar>
+void IntegratorObserverSubcycling<Scalar>::
+observeStartIntegrator(const Integrator<Scalar>& integrator){
+
+  const Teuchos::RCP<Teuchos::FancyOStream> out = integrator.getOStream();
+  Teuchos::OSTab ostab(out,0,"ScreenOutput");
+  *out << "\n    Begin Subcycling -------------------------------------------------------\n";
+    // << "  Step       Time         dt  Abs Error  Rel Error  Order  nFail  dCompTime"
+    // << std::endl;
```

Comment:
are these comment still needed?

Reply:
Probably not.  But I will need to make the subcycling output controllable from the input, because right now it is overly verbose but helpful for debugging.